trust-store-trust-policy-specification.md

```diff
@@ -135,6 +154,13 @@ Value descriptions
 - **`warn`**: This means implementation MUST perform the validation and if validation fails(because of any reason) the implementation MUST log an error and MUST NOT fail validation.
 - **`skip`**: This means implementation MUST NOT perform the validation.
 
+#### Scope Constraints
+
+- There MUST only be one trust policy applicable to an artifact.
+- There MUST not be two trust policies with the same scope.
+- There MUST only be only one trust policy without the `scope` key, which means that the trust policy has global scope(applicable to all artifacts).In the case of overlapping scopes, the longest prefix match rule is used.  For example, the `wabbit-networks.io/software/dotnet/ocp-release` image if there are two trust policies with overlapping scopes `registry.wabbit-networks.io/software` and `registery.wabbit-networks.io/software/dotnet` then the policy with the longest-prefix matching scope i.e. `registery.wabbit-networks.io/software/dotnet` will be used for signature evaluation.
```

Comment:
With the possibility of wildcards and the question on whether we really want to limit to one policy per scope, this requirement may go away.